### PR TITLE
Frontend styles

### PR DESCRIPTION
**Pull request by scziskh**

### CODE REVIEW

File: frontend/src/components/form/field.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { createFormControl } from "solid-forms";
+import { mergeProps, splitProps } from "solid-js";
+import ErrorMessage from "./error-message";
+
+const Field = (props) => {
```
Comment (ShkarupaDC) [inline frontend/src/components/form/field.jsx]: Please rename it to `TextInput`

File: frontend/src/components/form/button.jsx [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
       <Field {...buttonProps}>{ownProps.placeholder}</Field>
-    </div>
+    </>
   );
 };
 
 export default Button;
 
-const Field = styled('button')`
+const Field = styled("button")`
```
Comment (ShkarupaDC) [inline frontend/src/components/form/button.jsx]: Please rename it to `ButtonHTML` or something like that (maybe, search how it is done in the industry)

File: frontend/src/components/form/field.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        onInput={(event) => {
+          control().setValue(event.target.value);
+        }}
+        disabled={control().isDisabled}
+      />
+      <label for={props.name}>{props.placeholder}</label>
+      <ErrorMessage
+        isValid={props.control.isValid}
+        message={props.control.errors?.message}
+        withoutMessage={props.withoutMessage}
```
Comment (ShkarupaDC) [inline frontend/src/components/form/field.jsx]: Why do we need this?

File: frontend/src/components/form/radio-input.jsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import { styled } from "solid-styled-components";
+
+const RadioInput = (props) => {
+  props = mergeProps({ control: createFormControl("") }, props);
+  const [ownProps, inputProps] = splitProps(props, ["control"]);
+  const control = () => ownProps.control;
+
+  return (
+    <Wrapper>
+      <input
```
Comment (ShkarupaDC) [inline frontend/src/components/form/radio-input.jsx]: Why can't we use `Field` here?

File: frontend/src/components/header/index.jsx [hunk 5]
```diff
@@ -2,7 +2,7 @@ import { styled } from "solid-styled-components";
 
 const Header = (props) => {
   return (
-    <Wrapper>
+    <Wrapper {...props}>
```
Comment (ShkarupaDC) [inline frontend/src/components/header/index.jsx]: Should text be also passed here to `Wrapper`?

File: frontend/src/components/sections/edit-mask-section.jsx [hunk 6]
```diff
@@ -89,7 +93,7 @@ const EditMaskSection = () => {
         inExperimentId.value,
         masks,
         cameras(),
-        region.value === 'inner',
+        region.value === "inner"
```
Comment (ShkarupaDC) [inline frontend/src/components/sections/edit-mask-section.jsx]: Please, revert the change in quotes. Look carefully at the `.prettierrc` config. Here we have 
```json
"singleQuote": true
```

File: frontend/src/components/sections/compose-section.jsx [hunk 7]
```diff
@@ -104,74 +104,60 @@ const ComposeSection = () => {
...
-            )}
-          </For>
-          <TextInput
-            name={`experimentId`}
-            placeholder={`Experiment Id`}
-            control={outExperimentId}
-          />
-          <BottomWrapper>
+          <Show when={!!experiments.value.length}>
+            <Header text="Experiments" sidebar />
```
Comment (ShkarupaDC) [inline frontend/src/components/sections/compose-section.jsx]: Move all names to the left

File: frontend/src/components/sections/edit-mask-section.jsx [hunk 8]
```diff
@@ -115,9 +119,8 @@ const EditMaskSection = () => {
   };
 
   return (
-    <>
-      <Header text="Edit scene" />
-      <Wrapper>
+    <SectionLayout header={`Edit scene (Mask)`}>
```
Comment (ShkarupaDC) [inline frontend/src/components/sections/edit-mask-section.jsx]: Please change name to `Edit scene with image masks`

File: frontend/src/components/sections/edit-section.jsx [hunk 9]
```diff
@@ -52,75 +53,67 @@ const EditSection = () => {
 
   return (
     <>
-      <Header text="Edit scene" />
+      <Header text="Edit scene AABB" />
```
Comment (ShkarupaDC) [inline frontend/src/components/sections/edit-section.jsx]: Please change name to `Edit scene with AABB`

Comment (ShkarupaDC) [inline frontend/src/components/sections/edit-section.jsx]: Why don't we use the `SceneLayout` here?

File: frontend/src/components/form/file-input.jsx [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    <Wrapper>
+      <Field
+        type="file"
+        {...props}
+        onChange={(event) =>
+          setPlaceholder(() => {
+            const files = event.target.files;
+            return files.length === 0
+              ? `Choose files`
+              : `Chose ${files.length} file${files.length === 1 ? `` : `s`}`;
```
Comment (ShkarupaDC) [inline frontend/src/components/form/file-input.jsx]: Typo (`Chosen`)

Comment (ShkarupaDC) [inline frontend/src/components/form/file-input.jsx]: Make a separate function in this component

File: frontend/src/components/form/radio-input.jsx [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+        type="radio"
+        {...inputProps}
+        id={inputProps.id ?? `${inputProps.name}-${inputProps.value}`}
+        value={inputProps.value ?? control().value}
+        onInput={(event) => {
```
Comment (ShkarupaDC) [inline frontend/src/components/form/radio-input.jsx]: We don't need braces here

File: frontend/src/components/form/field.jsx (hunk 1 above)
Comment (scziskh) [reply frontend/src/components/form/field.jsx]: We renamed it to `Input` and reusing it at every input

File: frontend/src/components/sections/edit-section.jsx (hunk 9 above)
Comment (scziskh) [reply frontend/src/components/sections/edit-section.jsx]: `SectionLayout` renamed to `SectionLayoutWithSidebar`

**Comment by scziskh** [comment]
// Bugs

Position of mesh is not correct
